src/components/CreateWallet.jsx

```diff
@@ -52,57 +21,25 @@ export default function CreateWallet({ currentWallet, startWallet }) {
     }
   }
 
-  const isCreated = currentWallet && currentWallet.name === createdWallet
-  const canCreate = !currentWallet && !isCreated
-
-  if (isCreated && alert?.seedphrase) {
-    return (
-      <rb.Alert variant="success">
-        <rb.Alert.Heading>Wallet created succesfully!</rb.Alert.Heading>
-        <p className="d-flex align-items-center my-3">
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width="32"
-            height="32"
-            fill="currentColor"
-            className="me-3"
-            viewBox="0 0 16 16"
-          >
-            <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM8 4a.905.905 0 0 0-.9.995l.35 3.507a.552.552 0 0 0 1.1 0l.35-3.507A.905.905 0 0 0 8 4zm.002 6a1 1 0 1 0 0 2 1 1 0 0 0 0-2z" />
-          </svg>
-          <span>
-            Please write down your seed phrase and password!
-            <br />
-            Without this information you will not be able to access and recover your wallet!
-          </span>
-        </p>
-        <p>
-          Seed phrase: <strong>{alert.seedphrase}</strong>
-        </p>
-        <p className="mb-0">
-          Password: <strong>{alert.password}</strong>
-        </p>
-      </rb.Alert>
-    )
-  } else {
-    return canCreate ? (
+  return (
+    <>
       <rb.Form onSubmit={onSubmit} validated={validated} noValidate>
-        <h1>Create Wallet</h1>
-        {alert && <rb.Alert variant={alert.variant}>{alert.message}</rb.Alert>}
-        <rb.Form.Group className="mb-3" controlId="walletName">
+        <rb.Form.Group className="mb-4" controlId="walletName">
           <rb.Form.Label>Wallet Name</rb.Form.Label>
-          <rb.Form.Control name="wallet" style={{ maxWidth: '20em' }} required />
+          <rb.Form.Control name="wallet" placeholder="Your wallet..." required />
```

Comment:
```suggestion
          <rb.Form.Control name="wallet" placeholder="Your wallet..." disabled={isCreating} required />
```

It takes a few seconds till a wallet is created. A user should not be able to add more to the input while the wallet is in creation. However, I do not know if this is a user-friendly way to solve this. What do you think?

Reply:
Sure that's a very good point. The request to create the wallet is already sent off at that point so it wouldn't change anything technically but definitely a good addition from a UX perspective imo! 👌